rep-0105.rst

```diff
@@ -54,6 +54,9 @@ The ``map`` frame is useful as a long-term global reference, but
 discrete jumps make it a poor reference frame for local sensing and
 acting.
 
+It is recommended to align the map with east, north, up (ENU) as a default for consisent viewing, but is not required.
```

Comment:
What I'd like to do is make sure that there's a consistent orientation of maps by default if you view two maps separately they'll be oriented the same way by default. (Like most roadmaps in the US are printed with north up). If you hold two maps up side by side you can infer a lot from what side up the labels are drawn. 

I'll reply about the Z levels in the discourse thread.